apps/air-discount-scheme/backend/src/app/modules/flight/flight.controller.ts

```diff
@@ -373,6 +373,7 @@ export class PrivateFlightController {
   }
 
   @Get('users/:nationalId/flights')
+  @Scopes(AirDiscountSchemeScope.admin, AirDiscountSchemeScope.default)
```

Comment:
Is this API designed for admins or normal users?

I would guess it's for admins since it looks up PII of other users. So it should allow  the admin scope only.